src/components/ConversationWindow.js

```diff
@@ -557,20 +591,26 @@ class ConversationWindow extends PureComponent {
       } else {
         headerActionButton = <div id="ch_conv_block" onClick={() => this.blockUser()}>Block User</div>
       }
+
+      if(conversation.ban && Object.keys(conversation.ban).length) {
+        composerDisabled = true;
+        disableComposerMessage = LANGUAGE_PHRASES.BAN_USER_MESSAGE
+      }
+
+      // Store conversation admins for easy comparision later
+      if (conversation.members && conversation.members.length) {
+        conversationAdmins = conversation.members
+        .filter(member => member.isAdmin === true)
+        .map(member => member.userId);
+      }
     }
 
     const typingStrings = typingString(typing);
-    
-    // Store conversation admins for easy comparision later
-    let conversationAdmins = [];
-    if (conversation && conversation.members.length) {
-      conversationAdmins = conversation.members
-      .filter(member => member.isAdmin === true)
-      .map(member => member.userId);
-    }
 
-		return (
-  		<div id="ch_conv_window" className="ch-conv-window">
+    const showTabs = !connecting && !messageLoading && conversation && conversation.isAdmin && conversation.type && ["open", "public"].includes(conversation.type);
```

Comment:
@pardeep-kumar-bs why are we taking `!connecting` instead of `connected` check? Moreover ca't we move this in above if condition.

Reply:
Done.